### PR TITLE
initial comit

### PR DESCRIPTION
**Pull request by theVivek-kumar**
slider done

### CODE REVIEW

File: slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    --BLUE-20: #d9edf7;
+    --BACKGROUND_COLOR: #1a202c;
+    --BACKGROUND_COL-2CHOICE: #1a202c;
+    --WHITE: #ffffff;
+    --GRAY-80: #333333;
+    --GREY-60: #929191;
+    --GRAY-40: #8f8e8e;
+    --GRAY-50: #7e7a7a;
+    --GRAY-20: #C4C4C4;
+    --border10: rgb(214, 214, 214);
```
Comment (pankaj-prog) [inline slider/slider.css:33]: all these variables are declared in style.css then what's the point of declaring them again here ?

File: slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    color: var(--WHITE);
+   
+    
+}
+.slider-values ul{
+    display: flex;
+    gap: 7rem;
```
Comment (pankaj-prog) [inline slider/slider.css:63]: Suggestion: use justify-content space between to show the  values. 
It is flexible and can be used with components of diff widths

**Review by pankaj-prog — APPROVED** [review]

**Comment by Shinde-Sandesh** [comment]
You can remove some spaces in the code. Good to go

**Review by Shinde-Sandesh — APPROVED** [review]
You can remove some spaces in the code. Good to go

**Pull request merged**